components/login.js

```diff
@@ -61,7 +61,7 @@ export default function Login ({ providers, callbackUrl, error, Header }) {
       <div className={styles.login}>
         {Header && <Header />}
         <div className='text-center font-weight-bold text-muted pb-4'>
-          Not registered? Just login, we'll automatically create an account.
```

Comment:
What are you thoughts on the hazard of removing this? 

This was communicating something that isn't communicated in the new copy.